content/source/docs/enterprise/system-overview/security-model.html.md

```diff
@@ -0,0 +1,103 @@
+
+### Terraform Enterprise Requires You to Manage and Secure the Underlying Network and Infrastructure
+
+Infrastructure admins are required to manage all aspects of the underlying infrastructure. This includes initial provisioning, secure configuration, access control, network ACL configuration, and OS-level software updates. Terraform Enterprise cannot ensure the security of your data if the underlying infrastructure is compromised.
+
+### You are Responsible for Updating Your Terraform Enterprise Deployment
+
+We release security fixes, application features, and bug fixes for Terraform Enterprise each month. Infrastructure admins are responsible for applying updates.
+
+### You are Responsible for Availability, Backups, and Disaster Recovery
+
+Infrastructure admins are responsible for all aspects of reliability and availability. Refer to Terraform Enterprise documentation on [monitoring](../admin/monitoring.html), [backups and restores](../admin/backup-restore.html), and [high availability mode (active/active)](../admin/active-active.html) for more guidance on this topic.
+
+### Terraform Enterprise Isolates Terraform Operations via Docker Containers
+
+Unlike Terraform Cloud, Terraform Enterprise executes all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services, but may have access to resources available on hosts accessible from the your Terraform Enterprise instance. 
+
+## Recommendations for Securely Operating Terraform Enterprise
+
+In addition those provided in the [Terraform Cloud security model](../../cloud/architectural-overview/security-model.html), we recommend the following for Terraform Enterprise users. 
+
+### Run Terraform Enterprise in an Isolated Network, Limit Ingress Ports, and Restrict Access to Underlying Infrastructure
+
+To minimize attack surface, we recommend running Terraform Enterprise in an isolated network and limiting ingress ports to only 80 and 443. 
+
+For standalone deployments, port 8800 is reserved for the [Replicated admin console](../admin/admin-access.html), which is used for configuring Terraform Enterprise. This port should only be exposed to infrastructure admins. If you choose to configure Terraform Enterprise with the [automated process](../install/automating-the-installer.html), you can disable the Replicated admin console by passing the `disable-replicated-ui` argument to the installation script:
+
+```sudo bash ./install.sh disable-replicated-ui```
+
+Additionally, we recommend restricting access to the nodes that are running Terraform Enterprise. Terraform Enterprise can not ensure the security or integrity of its data if the underlying infrastructure is compromised.
```

Comment:
We've said this above. Keep this if you think it's worth saying again (it could be valuable to emphasize), but then please just change "its data" to "your data".

```suggestion
Additionally, we recommend restricting access to the nodes that are running Terraform Enterprise.
```

Reply:
I think its valuable to emphasize this. I'll update it to say "your data"